test/evmc/evmc.h

```diff
@@ -1181,7 +1181,7 @@ struct evmc_vm
  *
  * @par Binaries naming convention
  * For VMs distributed as shared libraries, the name of the library SHOULD match the VM name.
- * The convetional library filename prefixes and extensions SHOULD be ignored by the Client.
+ * The conventional library filename prefixes and extensions SHOULD be ignored by the Client.
```

Comment:
Note that this file comes from https://github.com/ethereum/evmc. It will be overwritten the next time we update it, so it should really be fixed upstream if we want the fix to stick.

Reply:
https://github.com/ethereum/evmc/pull/704